### PR TITLE
debug/workflow_schedule

### PR DESCRIPTION
**Pull request by chris-yim**
Adding retries to process_label to fix HTTPs Error and also adding logging of memory usage

### CODE REVIEW

File: labelbox/data/serialization/coco/instance_dataset.py [hunk 1]
```diff
@@ -123,6 +124,7 @@ def bbox_to_common(class_annotations: COCOObjectAnnotation,
     return annotations
 
 
+@retry()
```
Comment (b-delfino) [inline labelbox/data/serialization/coco/instance_dataset.py]: can you add stop=stop_after_attempt(20)

Comment (b-delfino) [reply labelbox/data/serialization/coco/instance_dataset.py]: and wait=wait_fixed(2)

Comment (b-delfino) [reply labelbox/data/serialization/coco/instance_dataset.py]: But also- I don't think this is gonna work. The retry needs to decorate the function that makes the http call, not this big one. So I assume get_image?

Comment (chris-yim) [reply labelbox/data/serialization/coco/instance_dataset.py]: its not get_image tho, the fail message was for calling the labelbox api and fails with getting a certain annotation

**Review by b-delfino — APPROVED** [review]

**Pull request merged**